frontend/src/scenes/saved-insights/savedInsightsLogic.ts

```diff
@@ -0,0 +1,55 @@
+import { kea } from 'kea'
+import api from 'lib/api'
+import { toParams } from 'lib/utils'
+import { DashboardItemType } from '~/types'
+import { savedInsightsLogicType } from './savedInsightsLogicType'
+
+interface InsightsResult {
+    results: DashboardItemType[]
+    count: number
+    previous?: string
+    next?: string
+}
+
+export const savedInsightsLogic = kea<savedInsightsLogicType<InsightsResult>>({
+    loaders: ({ values }) => ({
+        insights: {
+            __default: null as InsightsResult | null,
```

Comment:
I think having the default follow a consistent format like `{results: [], count: 0}` will make it easier down the line for null/empty checking in the component